.github/workflows/intelligence-docs.yml

```diff
@@ -99,9 +99,35 @@ jobs:
           name: swift-docs
           path: intelligence/docs/source/swift-api-ref/
 
+  build_kt_docs:
+    runs-on: ubuntu-22.04
+    name: Build Kotlin Docs
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Bootstrap
+        uses: ./.github/actions/bootstrap
+
+      - name: Set up JDK
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin'
+          java-version: '17'
+
+      - name: Build Kotlin docs
+        run: |
+          chmod +x ./intelligence/dev/build-kt-docs.sh
```

Comment:
@danielnugraha This should be run locally and committed, we shouldn't have to run it everytime